README.md

```
@@ -58,23 +58,34 @@ To circumvent this, FlareSolverr can be used to bypass Cloudflare protection. Fl
-<!---->
+Configuration is done via the Home Assistant UI. When adding the integration, you will be presented with several search options:
+
+*   **Manual**: Enter the GasBuddy Station ID directly.
+*   **Search by Home**: Uses your Home Assistant zone coordinates to find nearby stations.
+*   **Search by Postal Code**: Enter a specific Zip or Postal Code to find stations in that area.
+
+## Services
+
+The following services are available:
+
+| Service | Description | Arguments |
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: no_leading_or_trailing; Actual: leading_and_trailing; Unexpected leading pipe

(MD055, table-pipe-style)